.github/workflows/jetbrains-update-plugin-platform.yml

```diff
@@ -0,0 +1,21 @@
+name: JB Plugins Platform Update
+on:
+    workflow_dispatch:
+    schedule:
+        # At 11:00 on every day-of-week from Monday through Friday.
+        - cron: "0 11 * * 1-5"
+jobs:
+    update-backend-plugin-platform:
+        uses: ./.github/workflows/jetbrains-update-plugin-platform-template.yml
+        with:
+            pluginName: JetBrains Backend Plugin
+            pluginId: backend-plugin
+            xpath: "(/html/body/table[preceding::h2/text()='com.jetbrains.intellij.idea'][1]/tbody/tr/td[contains(text(),'-EAP-CANDIDATE-SNAPSHOT') and contains(text(),'222')]/text())[1]"
```

Comment:
Why 222 hardcoded?

Reply:
Good question! I actually forgot it there! Let me refactor it.
I'll extract the major version of the current platform and set it in a variable to use there as a guard statement to avoid creating PRs with major version regression (e.g. 222 -> 221) or upgrade (e.g. 222 -> 223).

Reply:
Done!